src/midiEditorCommands.cpp

```diff
@@ -479,7 +511,9 @@ class MidiEventIterator {
 	}
 
 	protected:
-	int getCount() const;
+	int getCount() const {
```

Comment:
We don't seem to use this anywhere except the iterator's own constructor. Can we get rid of this and call `EventType::getCount` directly in the constructor?